doc/source/tutorial-series-use-a-federated-learning-strategy-pytorch.ipynb

```diff
@@ -40,7 +42,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install -q flwr[simulation] torch torchvision"
+    "!pip install -q flwr-nightly[simulation] flwr-datasets[vision] torch torchvision"
```

Comment:
I wonder why do we need flwr-nightly here.

Reply:
I'll remove this after 1.9 release.